spec.html

```diff
@@ -23527,7 +23527,8 @@
     <!-- es6num="19.3.3" -->
     <emu-clause id="sec-properties-of-the-boolean-prototype-object">
       <h1>Properties of the Boolean Prototype Object</h1>
-      <p>The Boolean prototype object is the intrinsic object %BooleanPrototype%. The Boolean prototype object is an ordinary object. It is not a Boolean instance and does not have a [[BooleanData]] internal slot.</p>
```

Comment:
Objects which use the ordinary internal methods from [9.1](http://tc39.github.io/ecma262/#sec-ordinary-object-internal-methods-and-internal-slots) are called ["Ordinary objects" (4.3.6)](http://tc39.github.io/ecma262/#sec-ordinary-object). This does not mean ordinary objects are not allowed to have slots, otherwise it wouldn't make sense to have an [ObjectCreate (9.1.13)](http://tc39.github.io/ecma262/#sec-objectcreate) method to create ordinary objects with custom slots.